<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGetBorrow</class>
 <widget class="QWidget" name="FormGetBorrow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выдать/ сдать/ получить</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/images.qrc">
    <normaloff>:/form.jpeg</normaloff>:/form.jpeg</iconset>
  </property>
  <widget class="QPushButton" name="buttonPushRequest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Push</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modeRequest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string comment="Получение пользователем">get</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>borrow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>extension</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="bookInfoComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="line_bookID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_userID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book_NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User_NAME</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBookName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboUserName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book_ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User_ID</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
